### ISSUE TITLE
Resolution issues and songs not loading part way through song.

### ISSUE THREAD

**Issue by TheFantasticLoki** [fluddokt/opsu#276]
Since I updated to the latest version (0.16.0b->0.16.1a) my screen has gone back to 16:9 resolution but now at the top and bottom of the screen some circles will render halfway off screen. I have also encountered an issue that was occurring in the previous version as well. The song "Exit This Earth's Atomosphere (Camellia's ''PLANETARY//200STEP'' Remix)" Mapped by ProfessionalBox will stop rendering anything after a certain point in the song not long after the start. I have not encountered this in any other song to the date of this post. I would also like to request a feature to be added to the android version, Play area offset for phones such as the note 8 or any phone with a curved screen edge as to not have our pens slide off the screen messing up our plays.

**Comment by TheFantasticLoki** [comment]
Also would really like the Play area offset especially for the song I mentioned above

**Comment by fishporange** [comment]
There is an option to scale the play area, I have mine on 1x, it never touched the edge of my screen, however having it on the original does makes the circle sometimes get off screen.

It would be nice if opsu supports 18:9 displays, kinda annoying having a black bar on the side..

**Comment by TheFantasticLoki** [comment]
Yes and I did find out that when I updated my opsu application it changed
my scale to 2x from the previous 1x. But before updating mine was running
in 18;9 aspect ration and I do not know how I made it do so but I would
also like the ability to change the default resolution to what my screen
supports on my own accord.

**Comment by KeenanBurns** [comment]
I have the same issue as you. After the spinner at the start on Exit This Earth's Atomosphere, my screen immediately goes black. It also does this on certain other maps. The song "The Solace of Oblivion" also does this (for obvious reasons if you've seen the map), which is ironic because ProfessionalBox also mapped this song. I can see why The Solace of Oblivion would cause errors, but Exit This Earth's Atomosphere should be nearly as intensive as that song. 
Also, I play on a Samsung Galaxy Tab S 10.5
